caracal/workers/selfcal_worker.py

```diff
@@ -444,7 +444,7 @@ def change_header_and_type(filename, headfile, copy_head):
     def fake_image(trg, num, img_dir, mslist, field):
         key = 'image'
         key_mt = 'calibrate'
-
+        thread = config[key]['ncpu_img'] if config[key]['ncpu_img'] else ncpu
```

Comment:
Maybe I'm being pedantic, but *threads* or *nthread* would be better variable names

Reply:
Afraid so. See the comments from @o-smirnov above - the parameter is pretty much like the ncpu parameter so calling it ncpu_img makes sense.

That being said, I am happy to go with the consensus on this, whatever it is.

Reply:
I meant change the variable `thread` in line 447, not `ncpu_img`. But now that you mention it, it's better to also call the variable `ncpu_img` to be consistent.

Reply:
Ah, ok. Definitely being a bit pedantic there. 😂

Reply:
Will amend.

Reply:
This is done now.